src/components/Card/index.tsx

```diff
@@ -38,6 +38,11 @@ export interface CardProps {
      * Determine whether to highlight the card when mouse pointer is hovered on the card.
      */
     withHover?: boolean;
+    /**
+     * These props will be forwarded to the title
+     * (as long as disableTypography is not `true`).
```

Comment:
We did not expose the disableTypography. So pls remove that to avoid confusion.

Reply:
ok this should be solved